lib/svrquery/protocol/titanfall/types.go

```diff
@@ -139,15 +154,24 @@ func (a HealthFlags) DOS() bool {
 
 // BasicInfo represents basic information contained in a query response.
 type BasicInfo struct {
-	Port            uint16
-	Platform        string
-	PlaylistVersion string
-	PlaylistNum     uint32
-	PlaylistName    string
-	NumClients      byte
-	MaxClients      byte
-	Map             string
-	PlatformPlayers map[string]byte
+	Port                      uint16
+	Platform                  string
+	PlaylistVersion           string
+	PlaylistNum               uint32
+	PlaylistName              string
+	NumClients                byte
+	NumBotClients             byte
+	MaxClients                byte
+	TotalClientsConnectedEver uint32
+	Map                       string
+	PlatformPlayers           map[string]byte
+}
+
+// PerformanceInfoV9 represents frame information contained in a query response.
+type PerformanceInfoV9 struct {
+	PerformanceInfo
+	CommitMemory   uint32
+	ResidentMemory uint32
```

Comment:
Just double checking that these should be `uint32` and not `uint64` as for a 64bit server I would expect the latter?

Reply:
We're going off a schema they've provided us, so we're deferring to them (schema: https://docs.google.com/spreadsheets/d/1lY6i9EUoCOd8NnxT_OhsQeScCtx-vXqcU-s9Tc77nJ8/edit#gid=0). They've specified uint32 so to ensure we probably parse the packet we need to adhere to it.